app/Http/Controllers/BookingController.php

```diff
@@ -146,4 +146,46 @@ public function selectRoomAvailable(Request $request)
             return redirect()->route('booking');
         }
     }
+
+    public function saveInfo(Request $request)
+    {
+
+        if (Auth::check()) {
+            $user = User::find(Auth::id());
```

Comment:
```suggestion
            $user = Auth()->user();
```
em lại đi vòng vòng rồi